R/cdisc_data.R

```diff
@@ -33,7 +33,7 @@
 #'   )
 #' )
 cdisc_data <- function(...,
-                       join_keys = teal.data::join_keys(),
+                       join_keys = teal.data::join_keys_cdisc(...),
```

Comment:
`cdisc_data` but `join_keys_cdisc`. Once `cdisc_` is a prefix, once `_cdisc` is a suffix. Let's keep it as prefix, unless you have different view on this

Reply:
Function was renamed!